lib/confex_consul/local_cache.ex

```diff
@@ -87,4 +87,11 @@ defmodule ConfexConsul.LocalCache do
   defp refresh_cache_by_consul_key({{:via, ConfexConsul}, _type, consul_key, default_value}) do
     refresh_cache_by_consul_key({{:via, ConfexConsul}, consul_key, default_value})
   end
+
+  defp maybe_put_default_value(consul_key, default_value) do
```

Comment:
It's better to have some comments to describe why you used `previous value` when getting consul kv failed. :)

Reply:
Updated PR comment.